test/unittests/mongoBackend/mongoContextProvidersQuery_test.cpp

```diff
@@ -563,11 +564,37 @@ TEST(mongoContextProvidersQueryRequest, noPatternAttrsAll)
   /* Check response is as expected */
   EXPECT_EQ(SccOk, ms);
 
-  EXPECT_EQ(SccFound, res.errorCode.code);
-  EXPECT_EQ("Found", res.errorCode.reasonPhrase);
-  EXPECT_EQ("http://cr1.com", res.errorCode.details);
+  EXPECT_EQ(SccNone, res.errorCode.code);
```

Comment:
Yes, I know your opinion about this. I just don't agree ... :-D
Never mind that discussion for now.

Now, SccNone is returned and should be treated as an internal error and the upper layer must return 500 ...
Because here, 'case 2' will be detected.
See my confusion?
Or, have I misunderstood again :-) ?

And of course, we will not discuss this philosophic question as part of this PR.